tests/pytest/enrollment/test_views.py

```diff
@@ -113,15 +113,20 @@ def test_index_eligible_post_valid_form_failure(mocker, client, card_tokenize_fo
 
 @pytest.mark.django_db
 @pytest.mark.usefixtures("mocked_session_agency", "mocked_session_verifier", "mocked_session_eligibility")
-def test_index_eligible_post_valid_form_success(mocker, client, card_tokenize_form_data, mocked_analytics_module):
+def test_index_eligible_post_valid_form_success(
+    mocker,
+    client,
+    card_tokenize_form_data,
+    mocked_analytics_module,
+):
     mock_response = mocker.Mock()
     mock_response.success = True
     mocker.patch("benefits.enrollment.views.api.Client.enroll", return_value=mock_response)
 
     path = reverse(ROUTE_INDEX)
     response = client.post(path, card_tokenize_form_data)
 
-    mocked_analytics_module.returned_success.assert_called_once()
+    mocked_analytics_module.returned_success
     assert response.status_code == 200
     assert response.template_name == TEMPLATE_SUCCESS
     mocked_analytics_module.returned_success.assert_called_once()
```

Comment:
Why is line 129/132 repeated? I wasn't sure.

I was _trying_ to change the line to something more specific like:

```
mocked_analytics_module.returned_success.assert_called_once_with(app_request_enrollment_success)
```

But I kept getting an error like this:

```python
E       Args:
E       assert (<WSGIRequest...ess'>, '1234') == (<WSGIRequest...nt/success'>,)
E         At index 0 diff: <WSGIRequest: POST '/enrollment/success'> != <WSGIRequest: POST '/enrollment/success'>
E         Left contains one more item: '1234'
E         Full diff:
E         - (<WSGIRequest: POST '/enrollment/success'>,)
E         + (<WSGIRequest: POST '/enrollment/success'>, '1234')
```

and I couldn't figure out how to mock a request that `==` exactly like `(<WSGIRequest: POST '/enrollment/success'>, '1234')`. I would want to make sure that the `1234` group ID is correctly getting passed.

Reply:
cc @angela-tran @thekaveman

Reply:
Hmm not sure why that is repeated either, definitely shouldn't be!

Regarding your question, here's an example of checking that a specific call is in the list of calls for the spied function: https://github.com/cal-itp/eligibility-server/blob/main/tests/test_keypair.py#L52

The `mocker` fixture has a helper to create a [Call object](https://docs.python.org/3/library/unittest.mock.html#call), and you can look if that Call is inside the list of all Calls on the spy:

```python
assert mocker.call("arg1", "arg2", "etc.") in spy.call_args_list
```